adapters/tappx/tappx.go

```diff
@@ -51,14 +51,58 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	//In this section we create all EXT parameters
+	type VideoExt struct {
+		Rewarded int `json:"rewarded,omitempty"`
+	}
+	if tappxExt.Rewarded == 1 {
+		request.Imp[0].Video.Ext = json.RawMessage(`{"rewarded": 1}`)
+	}
```

Comment:
This new conditional requires json test coverage. You'll then see that the Video object is shared memory and must be copied first before making a change.

Reply:
Hi! A lot of thanks of this point. We now find useless to use the rewarded tappx param. We think that is better to tell to our publishers to send in video ext param directly like in all our oRTB connections. 😅

We clean all this in **a2ddc1f** commit: https://github.com/prebid/prebid-server/pull/1931/commits/a2ddc1f91aca8451a19aa4e651095b91338881dc